nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/pom.xml

```diff
@@ -117,6 +117,26 @@
             <version>1.19.0-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-translate</artifactId>
+            <version>1.12.328</version>
+        </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-polly</artifactId>
+            <version>1.12.328</version>
+        </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-transcribe</artifactId>
+            <version>1.12.328</version>
+        </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-textract</artifactId>
+            <version>1.12.328</version>
+        </dependency>
```

Comment:
Good point thank you. I checked it and the problem is around the super classes. I should refactor `AwsCredentialsProvider` related classes and we need to introduce version 2 compatible version of `AbstractAWSProcessor`, `AbstractAWSCredentialsProviderProcessor`, `AWSCredentialsProviderControllerService` etc. Should I go ahead and do this refactor as part of this PR?

Reply:
Thanks for the reply. It seems like it would be best to create a new Jira and pull request to create a new AWS Credentials Service based on version 2 of the SDK.

Reply:
On further review, moving forward with the current implementation based on SDK version 1 works, and subsequent refactoring for SDK version 2 can be handled once a new AWS Credentials Service is ready.